Userland/Libraries/LibArchive/Tar.cpp

```diff
@@ -24,10 +24,11 @@ unsigned TarFileHeader::expected_checksum() const
     return checksum;
 }
 
-void TarFileHeader::calculate_checksum()
+ErrorOr<void> TarFileHeader::calculate_checksum()
 {
     memset(m_checksum, ' ', sizeof(m_checksum));
-    VERIFY(DeprecatedString::formatted("{:06o}", expected_checksum()).copy_characters_to_buffer(m_checksum, sizeof(m_checksum)));
+    VERIFY(TRY(String::formatted("{:06o}", expected_checksum())).bytes_as_string_view().copy_characters_to_buffer(m_checksum, sizeof(m_checksum)));
```

Comment:
Isn't `VERIFY` specifically made to not have that issue/warning? The case with `assert` I get, but it seems like there isn't a reason to avoid this when using `VERIFY`, at least in my opinion.

Reply:
Yes, but the semantics is what I'm worried about, rather than whether the code will be accidentally ifdef'd out.